pilot/pkg/bootstrap/webhook.go

```diff
@@ -41,11 +41,14 @@ func (s *Server) initSecureWebhookServer(args *PilotArgs) {
 	log.Info("initializing secure webhook server for istiod webhooks")
 	// create the https server for hosting the k8s injectionWebhook handlers.
 	s.httpsMux = http.NewServeMux()
+	cipherSuites, _ := TLSCipherSuites(args.ServerOptions.TLSOptions.TLSCipherSuites)
```

Comment:
we probably shouldn't ignore the error?

Reply:
OK, i should look for another way to parse it outside and paas in the result.